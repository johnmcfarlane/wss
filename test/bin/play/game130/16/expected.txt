 64  1,9  | JA
 64  1,9  | JE
 64  1,9  | JO
 45 12,4  - YAJE
 39  5,6  - AJOG
 38  6,6  - JAG
 38  6,6  - JOG
 34  1,9  - JAR
 32  3,7  - JOE
 31  1,9  - JA
 28  6,14 - IJORE
 28 11,13 - JO
 28  3,13 | RAJ
 27  3,7  - JE
 27  3,7  - JO
 26  6,12 - AJAR
 26  4,6  - ORLAGE
 26  2,6  | REWARMS
 25  3,13 | JA
 25  3,13 | JE
 24  6,6  - LOGE
 24  7,10 | UREA
 23  8,13 - JALEO
 23  3,5  | JARA
 23  3,13 - JEEZ
 23  4,6  - OARAGE
 22  5,6  - ARAGE
 22 14,1  - JA
 22 14,1  - JE
 22 14,1  - JO
 22  5,6  - ORAGE
 22  4,4  - REJAIL
 21  5,7  | JE
 21  6,6  - RAGA
 21  6,6  - RAGE
 21 12,4  - YALE
 21 12,4  - YARL
 20  7,12 | LAO
 19  7,4  - JIAO
 18  7,6  - AGA
 18  7,6  - AGE
 18  7,6  - AGO
 18  7,13 | AOR
 18  6,4  - JAIL
 18  6,5  - JENOAR
 18 12,4  - YARE
 18 12,4  - YEAR
 18 12,4  - YORE
 17  7,12 | LA
 17 14,9  | QUEERER
 16  4,4  - AJARI
 16 10,13 - AROAR
 16 15,12 | DEJA
 16  5,4  | JALEE
 16  6,5  - JANE
 16 14,1  | JOLE
 16  5,5  | JOLE
 16 14,1  | JOOLA
 16  1,9  | LA
 16  3,7  - LEA
 16  3,7  - LER
 16  1,9  | LO
 16  3,7  - LOA
 16  3,7  - LOE
 16  3,7  - LOR
 16  8,10 | LOREAL
 15  7,12 | AA
 15  3,6  | AJAR
 15  5,5  - JEAN
 15  5,5  - JEON
 15  5,5  - JOAN
 15 14,1  | JOEL
 15  5,4  | JOREE
 15 12,6  | JOUL
 15  8,5  - NAJA
 15  3,4  - OLEARIA
 15 11,13 - ROAR
 15 11,13 - ROER
 15  6,13 - ZOEAL
 14  3,7  - AER
 14  5,4  - AERIAL
 14 13,5  | AJAR
 14  5,5  | AJEE
 14  3,13 | ALA
 14  3,7  - AOR
 14  1,15 - DEJA
 14  3,7  | JAR
 14 12,6  | JOUR
 14  6,6  - LAG
 14  6,6  - LOG
 14  8,10 | LORAL
 14  8,10 | LOREL
 14  5,4  - OARIAL
 14  3,13 | OLA
 14 14,2  | ORALE
 14  3,5  | RAJA
 14  6,13 - ZEAL
 14  6,13 - ZOAEA
 13  8,6  - GO
 13  5,6  | JEE
 13 12,6  | JEU
 13 14,1  | JOE
 13  5,6  | JOE
 13 12,7  | JUR
 13 12,7  - LARE
 13 12,7  - LEAR
 13 12,7  - LORA
 13 12,7  - LORE
 13 12,13 - OAR
 13 12,13 - OER
 13  9,6  | OX
 13  6,13 - ZERO
 13  6,13 - ZOEA
 13  6,13 - ZOL
 12  3,13 | AAL
 12  4,4  - AEOLIA
 12  9,13 - AERO
 12  7,4  - AILE
 12  4,4  - AROLIA
 12  7,4  - EILA
 12  5,8  | ELIA
 12  6,4  - JAI
 12 13,6  | JAR
 12  3,15 - JAR
 12  8,10 | LALO
 12 10,13 - LAO
 12  8,10 | LEAL
 12  8,10 | LEORA
 12  8,10 | LLER
 12 11,13 - LO
 12  8,10 | LOLA
 12  3,15 - RAJ
 12  4,4  - REALIA
 12  3,13 | REL
 12  7,4  - RIAL
 12  7,4  - RILE
 12  7,10 | URA
 12  6,13 - ZER
 12  6,13 - ZOA
 11  3,13 | AL
 11 10,13 - ARO
 11  3,13 | EL
 11  4,7  - EO
 11  4,7  - ER
 11  8,4  - ILA
 11  8,4  - ILE
 11  3,7  | JA
 11  3,15 - JA
 11  3,13 - JE
 11 14,1  | JO
 11  7,4  | LA
 11  3,14 | LA
 11  7,4  - LIAR
 11  7,4  | LO
 11  3,7  - LO
 11  9,3  | LOA
 11  9,3  | LOE
 11  4,7  - OE
 11  4,7  - OR
 11  6,6  - RAG
 11  6,4  - RAIAE
 11  6,6  - ROG
 11  6,13 - ZA
 10  1,9  | AA
 10 13,7  - AAL
 10  1,9  | AE
 10 13,4  | ALA
 10  6,12 - ALAE
 10  6,12 - ALAR
 10 13,4  | ALE
 10  6,12 - ALEA
 10  6,12 - ALOE
 10  3,5  | AREAL
 10 13,4  | AREAR
 10  5,2  | AREOLAE
 10  5,3  | AREOLE
 10  6,12 - ARLE
 10 13,4  | AROAR
 10 11,13 - EO
 10  5,8  | ERIA
 10 12,7  - LAO
 10 12,7  - LAR
 10  1,9  - LAR
 10  8,10 | LARE
 10 12,7  - LEA
 10  8,10 | LEAR
 10  6,5  - LENORA
 10 12,7  - LER
 10 12,7  - LOA
 10  5,5  - LOANER
 10 12,7  - LOE
 10 12,7  - LOR
 10  8,10 | LORA
 10  8,10 | LORE
 10 13,4  | OLA
 10 13,4  | OLE
 10  4,5  - OLEANA
 10  3,5  - ORLEAN
 10  3,13 | REA
 10  3,5  | REAAL
 10  3,5  - RELOAN
 10  5,4  - REOIL
  9  7,4  | AA
  9 13,4  | AAL
  9  1,9  - AAL
  9  7,4  | AE
  9  3,7  - AE
  9 13,4  | AL
  9  3,14 | AL
  9  5,5  - ALANE
  9  5,5  - ALONE
  9  7,5  - ANOLE
  9 13,13 - ARE
  9  6,4  - ARIL
  9 13,13 - ARO
  9 13,4  | EL
  9  3,14 | EL
  9  7,5  - ENROL
  9 13,13 - ERA
  9 12,7  - LA
  9  2,14 | LA
  9  6,5  - LANAO
  9  5,4  - LARIA
  9 12,7  - LO
  9  2,14 | LO
  9  6,5  - LONER
  9  7,4  | OE
  9  5,5  - OLENA
  9  6,4  - RAIL
  9  6,5  - RANAL
  9  6,5  - RENAL
  9  6,4  - ROIL
  9 13,11 | TEAL
  9 13,11 | TEEL
  9 12,11 | UPLA
  9  4,12 - VEAL
  8  6,12 - AAL
  8 13,4  | AER
  8  6,12 - AERO
  8  7,6  - AG
  8  7,4  - AIL
  8  5,14 | AL
  8 13,7  - ALA
  8  6,12 - ALA
  8 13,7  - ALE
  8  6,12 - ALE
  8  4,5  | ALEF
  8  6,4  - ALIA
  8  7,5  - ANAL
  8  7,6  | ANE
  8  7,5  - ANEAR
  8 13,4  | AOR
  8 13,4  | ARE
  8  6,12 - AREA
  8  5,5  - ARENA
  8  3,3  | AREOLA
  8 13,4  | ARO
  8 13,10 - AUL
  8 15,12 | DEAL
  8 13,4  | EAR
  8  5,4  | EARLE
  8  5,14 | EL
  8  6,4  - ELIA
  8  7,5  - ENOL
  8 13,4  | ERA
  8  3,6  | ERAL
  8  6,5  - LANA
  8  6,5  - LANE
  8  8,10 | LAO
  8  8,10 | LAR
  8  4,5  - LARON
  8  8,10 | LEA
  8  4,5  | LEAF
  8  4,5  - LEARN
  8  6,5  - LENA
  8  6,5  - LENO
  8  8,10 | LER
  8  7,5  - LNR
  8  8,10 | LOA
  8  4,5  | LOAF
  8  8,10 | LOE
  8  6,5  - LONE
  8  8,10 | LOR
  8  4,5  - LORAN
  8  4,5  - LOREN
  8 13,4  | OAR
  8 13,4  | OER
  8  7,4  - OIL
  8 13,7  - OLA
  8  4,5  | OLAF
  8 13,7  - OLE
  8 13,4  | ORA
  8  3,6  | ORAL
  8  5,4  | ORALE
  8 13,4  | ORE
  8  4,5  | RALF
  8  3,6  | REAL
  8 14,1  | ROLEO
  8  4,5  | ROLF
  8  4,12 - VOAR
  7 13,4  | AA
  7  3,13 | AA
  7  3,14 | AA
  7  1,9  - AAA
  7  3,7  | AAL
  7  4,5  - AARON
  7 13,4  | AE
  7  3,13 | AE
  7 13,7  - AER
  7  4,5  - AERON
  7  3,6  | ALAE
  7  5,5  - ALAN
  7  3,6  | ALAR
  7  5,5  - ALEN
  7  7,5  - ANOA
  7  7,5  - ANRE
  7 13,7  - AOR
  7 13,4  | AR
  7 13,7  - ARE
  7  4,5  - AREAN
  7  6,4  - ARIA
  7  5,5  | ARLE
  7  6,5  - ARNA
  7  6,5  - ARNE
  7 13,7  - ARO
  7  1,15 - DALAR
  7 15,12 | DEAR
  7 15,12 | DEL
  7 15,12 | DERO
  7 13,7  - EAR
  7  1,9  - EAR
  7 14,3  - EL
  7  5,5  - ELAN
  7  4,6  | ELF
  7  5,5  - ELON
  7 13,4  | EO
  7  2,14 | EO
  7  5,5  | EOLE
  7 13,4  | ER
  7 13,7  - ERA
  7  6,4  - ERIA
  7  1,9  - LA
  7  5,5  - LAEN
  7  5,5  - LARN
  7  5,5  - LEAN
  7 13,5  | LEAR
  7  3,6  | LEAR
  7  5,5  - LEON
  7  3,4  | LEORA
  7  5,5  - LOAN
  7  4,6  | LOF
  7  3,5  | LORA
  7  5,5  - LORN
  7  2,11 - MEA
  7  2,11 - MOA
  7  8,5  - NALE
  7  8,5  - NEAL
  7  8,5  - NOEL
  7 13,7  - OAR
  7  1,9  - OAR
  7 13,4  | OE
  7  3,13 | OE
  7  2,14 | OE
  7 13,7  - OER
  7 14,2  | OLEA
  7 14,2  | OLEO
  7 14,2  | OLOR
  7  7,5  - ONER
  7 13,4  | OR
  7 13,7  - ORA
  7 13,7  - ORE
  7  5,5  | ORLE
  7  6,5  - ORNA
  7  6,4  - RAIA
  7  5,5  | RALE
  7  6,5  - RANA
  7  6,5  - RANE
  7  3,13 | RE
  7  2,14 | RE
  7  6,5  - RENO
  7 14,1  | ROLE
  7  5,5  | ROLE
  7  6,5  - RONE
  7 13,10 - RUA
  7 13,10 - RUE
  7 13,11 | TEAR
  7 12,11 | UPO
  6  5,14 | AA
  6  6,12 - AAA
  6  5,14 | AE
  6  5,5  - AEON
  6  6,12 - AER
  6  3,8  | AL
  6  2,9  - AL
  6  5,5  | ALAE
  6 13,5  | ALAR
  6  5,6  | ALE
  6  3,5  | ALEA
  6  5,5  | ALEE
  6  5,5  | ALOE
  6 12,6  | ALUR
  6  7,5  - ANA
  6  7,5  - ANE
  6  6,12 - AOR
  6 12,6  | AOUL
  6  5,14 | AR
  6  6,12 - ARE
  6  5,4  | AREAE
  6  5,5  - AREN
  6  4,6  | ARF
  6  6,12 - ARO
  6 10,1  | ATOLE
  6  3,14 - AlLIE
  6 15,12 | DEA
  6 15,12 | DER
  6  1,15 - DOLA
  6  5,5  - EARN
  6 14,3  - EO
  6  5,5  - EOAN
  6 14,3  - ER
  6  5,14 | ER
  6  4,6  | ERF
  6 14,1  - LA
  6  6,5  - LAN
  6  3,7  | LAR
  6  5,5  | LARE
  6 12,6  | LAUR
  6  5,5  | LERE
  6  9,9  - LI
  6  9,9  | LI
  6 14,1  - LO
  6 14,1  | LORA
  6 14,1  | LORE
  6  5,5  | LORE
  6 14,1  | LORO
  6 12,6  | LOUR
  6  8,5  - NARE
  6  8,5  - NEAR
  6  8,5  - NOL
  6  8,5  - NORA
  6  8,5  - NORE
  6  4,6  | OAF
  6 14,3  - OE
  6  7,14 | OE
  6 14,2  | OLA
  6 14,2  | OLE
  6  5,6  | OLE
  6  3,5  | OLEA
  6  7,5  - ONA
  6  7,5  - ONE
  6 14,3  - OR
  6  7,14 | OR
  6  3,6  | ORAE
  6 14,2  | ORAL
  6  4,6  | ORF
  6 14,2  | ORLE
  6 14,2  | ORLO
  6  3,14 - OlLIE
  6 12,6  | RAUL
  6 14,3  - RE
  6  4,6  | REF
  6  7,4  - RIA
  6  7,4  - RIO
  6  5,6  | RLE
  6  5,5  - ROAN
  6 14,1  | ROOL
  5  3,7  | AAA
  5 13,7  - AL
  5  5,4  - ALAI
  5  5,4  - AOLI
  5 13,13 - AR
  5 13,5  | ARAR
  5  3,5  | AREA
  5  6,5  - ARN
  5 12,7  | AUL
  5 15,12 | DE
  5  1,15 - DORA
  5  3,7  | EAR
  5  4,7  | EF
  5 14,1  - EL
  5 13,7  - EL
  5  6,5  - EON
  5 13,13 - ER
  5  6,5  - ERN
  5  6,14 - ILEA
  5  8,4  - IO
  5  5,4  - LARI
  5  5,6  | LEE
  5 12,6  | LEU
  5  9,3  | LO
  5 14,1  | LOA
  5 14,1  | LOE
  5  5,6  | LOE
  5 14,1  | LOO
  5 14,1  | LOR
  5  5,4  - LORI
  5 12,6  | LOU
  5 12,7  | LUE
  5 12,7  | LUO
  5 12,7  | LUR
  5  8,5  - NAA
  5  8,5  - NAE
  5  8,5  - NAR
  5  8,5  - NEA
  5  8,5  - NEO
  5  8,5  - NOA
  5  8,5  - NOR
  5  3,7  | OAR
  5  4,7  | OF
  5 14,2  | ORAE
  5  5,5  | ORAE
  5  6,5  - RAN
  5 14,13 - RE
  5 13,5  | REAR
  5  6,5  - REN
  5 13,5  | ROAR
  5 13,5  | ROER
  5  6,5  - RON
  5 12,6  | ROUE
  5 13,11 | TEA
  5 13,11 | TEE
  5 10,2  | TOLA
  5 10,2  | TOLE
  5 10,2  | TOOL
  5 12,4  - YAR
  5 12,4  - YEA
  5 12,4  - YER
  5 12,4  - YOR
  5  4,14 - lEILA
  4 13,7  - AA
  4  3,8  | AA
  4  1,9  - AA
  4  2,9  - AA
  4  3,15 - AAL
  4 13,7  - AE
  4  3,8  | AE
  4  2,9  - AE
  4  9,9  - AI
  4  9,9  | AI
  4  3,6  | ALA
  4  7,5  - AN
  4 14,1  | AOR
  4 13,7  - AR
  4  3,8  | AR
  4  2,9  - AR
  4  5,6  | ARE
  4 12,6  | ARU
  4  3,14 - AlAI
  4  1,14 - EARl
  4 12,6  | EAU
  4  7,5  - EN
  4 14,1  - EO
  4 13,7  - EO
  4 14,1  - ER
  4 13,7  - ER
  4  1,14 - ERAl
  4  5,6  | ERE
  4  6,14 - IARE
  4  6,14 - ILA
  4  6,14 - ILE
  4  6,4  - LAI
  4  3,15 - LAO
  4 13,6  | LAR
  4  3,15 - LAR
  4  3,6  | LEA
  4  6,4  - LEI
  4 13,6  | LER
  4  8,10 | LL
  4  3,6  | LOA
  4 13,6  | LOR
  4 12,7  | LU
  4  3,14 - Ll
  4  8,5  - NA
  4  8,5  - NE
  4  8,5  - NO
  4 14,2  | OAR
  4 14,1  - OE
  4 13,7  - OE
  4 14,2  | OER
  4  9,9  - OI
  4  9,9  | OI
  4  3,6  | OLA
  4  7,5  - ON
  4 14,1  - OR
  4 13,7  - OR
  4 14,2  | ORA
  4 14,2  | ORE
  4  5,6  | ORE
  4 12,7  | OUR
  4  5,6  | REE
  4 14,1  | ROE
  4  5,6  | ROE
  4 14,1  | ROO
  4 12,7  | RUA
  4 12,7  | RUE
  4 10,2  | TOEA
  4 10,2  | TOL
  4 10,2  | TORA
  4 10,2  | TORE
  4 10,2  | TORO
  4 12,4  - YA
  4 12,4  - YE
  4 12,4  - YO
  3  3,6  | AAA
  3  3,15 - AAA
  3  5,7  | AE
  3 13,6  | AER
  3  6,12 - AL
  3  4,15 - AL
  3 13,6  | AOR
  3 13,6  | ARR
  3  1,15 - DA
  3  1,15 - DE
  3  1,15 - DO
  3 13,6  | EAR
  3  3,15 - EAR
  3  5,7  | EE
  3  1,14 - EL
  3  3,6  | ERA
  3 13,6  | ERR
  3  6,14 - IRA
  3  6,14 - IRE
  3  3,7  | LA
  3  8,10 | LA
  3  3,15 - LA
  3  7,4  - LI
  3 14,1  | LO
  3  8,10 | LO
  3 13,6  | OAR
  3  3,15 - OAR
  3 14,2  | OE
  3  5,7  | OE
  3 13,6  | OER
  3 14,2  | OO
  3 14,2  | OR
  3  3,6  | ORA
  3  5,7  | RE
  3  3,6  | REA
  3  6,4  - ROI
  3 10,2  | TOA
  3 10,2  | TOE
  3 10,2  | TOO
  3 10,2  | TOR
  3  4,14 - lAIR
  3  4,14 - lOIR
  2  3,7  | AA
  2  6,12 - AA
  2  3,15 - AA
  2  4,15 - AA
  2  6,12 - AE
  2  3,13 - AE
  2  4,15 - AE
  2  7,4  - AI
  2 13,7  | AR
  2  1,10 - AR
  2  6,12 - AR
  2  4,15 - AR
  2  3,14 - Al
  2  3,13 - EE
  2  1,14 - EE
  2 14,1  | EO
  2  1,14 - EO
  2 13,7  | ER
  2  1,10 - ER
  2  1,14 - ER
  2  3,14 - El
  2  6,14 - IO
  2  3,13 - OE
  2  7,4  - OI
  2 14,1  | OO
  2 13,7  | OR
  2  1,10 - OR
  2  3,13 - RE
  2  4,14 - lAI
  2  4,14 - lEI
