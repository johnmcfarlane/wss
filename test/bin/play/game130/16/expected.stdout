 64  1,9  | JA
 64  1,9  | JE
 64  1,9  | JO
 38  6,6  - JAG
 38  6,6  - JOG
 34  1,9  - JAR
 32  3,7  - JOE
 31  1,9  - JA
 28 11,13 - JO
 28  3,13 | RAJ
 27  3,7  - JE
 27  3,7  - JO
 26  6,12 - AJAR
 26  2,6  | REWARMS
 25  3,13 | JA
 25  3,13 | JE
 24  6,6  - LOGE
 23  3,13 - JEEZ
 22 14,1  - JA
 22 14,1  - JE
 22 14,1  - JO
 21  5,7  | JE
 21  6,6  - RAGA
 21  6,6  - RAGE
 19  7,4  - JIAO
 18  7,6  - AGA
 18  7,6  - AGE
 18  7,6  - AGO
 18 12,4  - YARE
 18 12,4  - YEAR
 18 12,4  - YORE
 17  7,12 | LA
 17 14,9  | QUEERER
 16 15,12 | DEJA
 16  6,5  - JANE
 16 14,1  | JOLE
 16  5,5  | JOLE
 16  1,9  | LA
 16  3,7  - LEA
 16  1,9  | LO
 16  8,10 | LOREAL
 15  7,12 | AA
 15  3,6  | AJAR
 15  5,5  - JEAN
 15  5,5  - JEON
 15 11,13 - ROAR
 15  6,13 - ZOEAL
 14 13,5  | AJAR
 14  5,5  | AJEE
 14  1,15 - DEJA
 14  3,7  | JAR
 14  6,6  - LAG
 14  6,6  - LOG
 14  8,10 | LORAL
 14  3,5  | RAJA
 14  6,13 - ZEAL
 13  8,6  - GO
 13  5,6  | JEE
 13 12,6  | JEU
 13 14,1  | JOE
 13  5,6  | JOE
 13 12,13 - OAR
 13  9,6  | OX
 13  6,13 - ZERO
 13  6,13 - ZOEA
 12  3,13 | AAL
 12  9,13 - AERO
 12  6,4  - JAI
 12 13,6  | JAR
 12  3,15 - JAR
 12  8,10 | LEAL
 12 11,13 - LO
 12  3,15 - RAJ
 12  4,4  - REALIA
 12  6,13 - ZOA
 11  4,7  - EO
 11  4,7  - ER
 11  3,7  | JA
 11  3,15 - JA
 11  3,13 - JE
 11 14,1  | JO
 11  7,4  | LA
 11  7,4  - LIAR
 11  7,4  | LO
 11  3,7  - LO
 11  4,7  - OE
 11  4,7  - OR
 11  6,6  - RAG
 11  6,13 - ZA
 10  1,9  | AA
 10 13,7  - AAL
 10  1,9  | AE
 10 13,4  | ALA
 10  6,12 - ALAE
 10  6,12 - ALAR
 10 13,4  | ALE
 10  6,12 - ALOE
 10  3,5  | AREAL
 10  5,2  | AREOLAE
 10  5,3  | AREOLE
 10 11,13 - EO
 10  4,4  - LAARI
 10  1,9  - LAR
 10  8,10 | LEAR
 10  5,5  - LOANER
 10  8,10 | LORE
 10 13,4  | OLE
 10  3,5  - RELOAN
  9  7,4  | AA
  9 13,4  | AAL
  9  1,9  - AAL
  9  7,4  | AE
  9  3,7  - AE
  9 13,4  | AL
  9  3,14 | AL
  9  5,5  - ALANE
  9  5,5  - ALONE
  9  7,5  - ANOLE
  9 13,13 - ARE
  9 13,4  | EL
  9  3,14 | EL
  9  7,5  - ENROL
  9 13,13 - ERA
  9  2,14 | LA
  9  2,14 | LO
  9  6,5  - LONER
  9  7,4  | OE
  9  6,5  - RENAL
  9 13,11 | TEAL
  9 13,11 | TEEL
  9  4,12 - VEAL
  8  6,12 - AAL
  8  6,12 - AERO
  8  7,6  - AG
  8 13,7  - ALA
  8  6,12 - ALA
  8 13,7  - ALE
  8  6,12 - ALE
  8  4,5  | ALEF
  8  7,5  - ANAL
  8  7,6  | ANE
  8  7,5  - ANEAR
  8 13,4  | ARE
  8  6,12 - AREA
  8  5,5  - ARENA
  8  3,3  | AREOLA
  8 15,12 | DEAL
  8 13,4  | EAR
  8  5,14 | EL
  8  7,5  - ENOL
  8 13,4  | ERA
  8  6,5  - LANE
  8  8,10 | LAR
  8  8,10 | LEA
  8  4,5  | LEAF
  8  4,5  - LEARN
  8  6,5  - LENO
  8  4,5  | LOAF
  8  6,5  - LONE
  8  4,5  - LORAN
  8 13,4  | OAR
  8 13,7  - OLE
  8 13,4  | ORA
  8  3,6  | ORAL
  8 13,4  | ORE
  8  3,6  | REAL
  8  4,5  | ROLF
  7 13,4  | AA
  7  3,13 | AA
  7  3,14 | AA
  7  3,7  | AAL
  7 13,4  | AE
  7  3,13 | AE
  7  3,6  | ALAE
  7  5,5  - ALAN
  7  3,6  | ALAR
  7  7,5  - ANOA
  7 13,4  | AR
  7 13,7  - ARE
  7  6,4  - ARIA
  7 15,12 | DEAR
  7 15,12 | DEL
  7 13,7  - EAR
  7  1,9  - EAR
  7 14,3  - EL
  7  5,5  - ELAN
  7  4,6  | ELF
  7 13,4  | EO
  7  2,14 | EO
  7 13,4  | ER
  7 13,7  - ERA
  7  1,9  - LA
  7  5,4  | LAREE
  7  5,5  - LEAN
  7 13,5  | LEAR
  7  3,6  | LEAR
  7  5,5  - LOAN
  7  5,5  - LORN
  7  2,11 - MOA
  7  8,5  - NALA
  7  8,5  - NOEL
  7 13,7  - OAR
  7  1,9  - OAR
  7 13,4  | OE
  7  3,13 | OE
  7  2,14 | OE
  7 14,2  | OLEA
  7 14,2  | OLEO
  7 13,4  | OR
  7 13,7  - ORA
  7 13,7  - ORE
  7  5,5  | ORLE
  7  6,4  - RAIA
  7  5,5  | RALE
  7  3,13 | RE
  7  2,14 | RE
  7 14,1  | ROLE
  7  5,5  | ROLE
  7 13,11 | TEAR
  7 12,11 | UPO
  6  5,5  - AEON
  6  3,8  | AL
  6  2,9  - AL
  6  5,5  | ALAE
  6 13,5  | ALAR
  6  5,6  | ALE
  6  5,5  | ALEE
  6  5,5  | ALOE
  6  7,5  - ANA
  6  7,5  - ANE
  6  6,12 - ARE
  6  5,4  | AREAE
  6  4,6  | ARF
  6  5,5  - EARN
  6 14,3  - EO
  6 14,3  - ER
  6  5,14 | ER
  6 14,1  - LA
  6  3,7  | LAR
  6  9,9  - LI
  6  9,9  | LI
  6 14,1  - LO
  6 14,1  | LORE
  6  5,5  | LORE
  6 12,6  | LOUR
  6  8,5  - NEAR
  6  4,6  | OAF
  6 14,3  - OE
  6 14,2  | OLE
  6  5,6  | OLE
  6  3,5  | OLEA
  6  7,5  - ONE
  6 14,3  - OR
  6 14,2  | ORAL
  6  4,6  | ORF
  6 14,2  | ORLE
  6 14,3  - RE
  6  4,6  | REF
  6  7,4  - RIA
  6  5,5  - ROAN
  5 13,7  - AL
  5 13,13 - AR
  5  3,5  | AREA
  5 15,12 | DE
  5  3,7  | EAR
  5  4,7  | EF
  5 14,1  - EL
  5 13,7  - EL
  5  6,5  - EON
  5 13,13 - ER
  5  6,5  - ERN
  5  6,14 - ILEA
  5  5,4  - LARI
  5  5,6  | LEE
  5 12,6  | LEU
  5 14,1  | LOO
  5  8,5  - NAE
  5  8,5  - NOR
  5  3,7  | OAR
  5  4,7  | OF
  5  6,5  - RAN
  5 14,13 - RE
  5 13,5  | REAR
  5 13,5  | ROAR
  5 12,6  | ROUE
  5 13,11 | TEA
  5 13,11 | TEE
  5 10,2  | TOLA
  5 10,2  | TOLE
  5 10,2  | TOOL
  5 12,4  - YAR
  5 12,4  - YEA
  4 13,7  - AA
  4  3,8  | AA
  4  1,9  - AA
  4  2,9  - AA
  4  3,15 - AAL
  4 13,7  - AE
  4  3,8  | AE
  4  2,9  - AE
  4  9,9  - AI
  4  9,9  | AI
  4  3,6  | ALA
  4  7,5  - AN
  4 13,7  - AR
  4  3,8  | AR
  4  2,9  - AR
  4  5,6  | ARE
  4  1,14 - EARl
  4 12,6  | EAU
  4  7,5  - EN
  4 14,1  - EO
  4 13,7  - EO
  4 14,1  - ER
  4 13,7  - ER
  4  5,6  | ERE
  4 13,6  | LAR
  4  3,15 - LAR
  4  3,6  | LEA
  4  6,4  - LEI
  4  8,5  - NA
  4  8,5  - NE
  4  8,5  - NO
  4 14,2  | OAR
  4 14,1  - OE
  4 13,7  - OE
  4  9,9  - OI
  4  9,9  | OI
  4  7,5  - ON
  4 14,1  - OR
  4 13,7  - OR
  4 14,2  | ORA
  4 14,2  | ORE
  4  5,6  | ORE
  4 12,7  | OUR
  4  5,6  | REE
  4 14,1  | ROE
  4  5,6  | ROE
  4 12,7  | RUE
  4 10,2  | TOEA
  4 10,2  | TORA
  4 10,2  | TORE
  4 10,2  | TORO
  4 12,4  - YA
  4 12,4  - YE
  4 12,4  - YO
  3  5,7  | AE
  3  6,12 - AL
  3  4,15 - AL
  3  1,15 - DA
  3  1,15 - DE
  3  1,15 - DO
  3 13,6  | EAR
  3  3,15 - EAR
  3  5,7  | EE
  3  1,14 - EL
  3  3,6  | ERA
  3 13,6  | ERR
  3  6,14 - IRE
  3  3,7  | LA
  3  8,10 | LA
  3  3,15 - LA
  3  7,4  - LI
  3 14,1  | LO
  3  8,10 | LO
  3 13,6  | OAR
  3  3,15 - OAR
  3 14,2  | OE
  3  5,7  | OE
  3 14,2  | OO
  3 14,2  | OR
  3  3,6  | ORA
  3  6,4  - RAI
  3  5,7  | RE
  3 10,2  | TOE
  3 10,2  | TOO
  3 10,2  | TOR
  3  4,14 - lAIR
  2  3,7  | AA
  2  6,12 - AA
  2  3,15 - AA
  2  4,15 - AA
  2  6,12 - AE
  2  3,13 - AE
  2  4,15 - AE
  2  7,4  - AI
  2 13,7  | AR
  2  1,10 - AR
  2  6,12 - AR
  2  4,15 - AR
  2  3,14 - Al
  2  3,13 - EE
  2  1,14 - EE
  2 14,1  | EO
  2  1,14 - EO
  2 13,7  | ER
  2  1,10 - ER
  2  1,14 - ER
  2  3,14 - El
  2  3,13 - OE
  2  7,4  - OI
  2 14,1  | OO
  2 13,7  | OR
  2  1,10 - OR
  2  3,13 - RE
  2  4,14 - lEI
